### PR TITLE
PUB-1517 & 1523 Service timeout for media users + Admin logout fixes 

### PR DESCRIPTION
**Pull request by KianKwa**
### JIRA link ###

https://tools.hmcts.net/jira/browse/PUB-1517

### Change description ###

- Fixed issue with service not timing out for media users after 1 hour inactivity.
- Destroy session during user log out.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by junaidiqbalmoj — APPROVED** [review]
Tested the PR and looks good to me :)

File: src/main/service/sessionManagementService.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import process from 'process';
+import moment from 'moment';
+import {checkRoles, verifiedRoles} from '../authentication/authenticationHandler';
+
+const logger = Logger.getLogger('session-management');
+const authenticationConfig = require('../authentication/authentication-config.json');
+const FRONTEND_URL = process.env.FRONTEND_URL || 'https://pip-frontend.staging.platform.hmcts.net';
+const defaultSessionExpiry = 60 * 60 * 1000; // default to 1 hour
+
+export class SessionManagementService {
```
Comment (ChrisS1512) [inline src/main/service/sessionManagementService.ts:12]: Extra logs should be removed hear to prevent spam

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: src/main/service/sessionManagementService.ts (hunk 1 above)
Comment (KianKwa) [reply src/main/service/sessionManagementService.ts:10]: Agree the logs look redundant. have removed :)

**Review by ChrisS1512 — DISMISSED** [review]

**Review by junaidiqbalmoj — DISMISSED** [review]

**Review by joshblackmoor — APPROVED** [review]

**Review by joshblackmoor — DISMISSED** [review]
👍

**Review by ChrisS1512 — DISMISSED** [review]

**Review by ChrisS1512 — APPROVED** [review]

**Pull request merged**